index.bs

```diff
@@ -1933,7 +1933,7 @@ SHOULD be aborted.
     See [WHATWG HTML WG Issue #2711](https://github.com/whatwg/html/issues/2711) for more details. 
 
 
-## Authentication Extensions Client Inputs (typedef <dfn>AuthenticationExtensionsClientInputs</dfn>) ## {#iface-authentication-extensions-client-inputs}
+## Authentication Extensions Client Inputs (typedef AuthenticationExtensionsClientInputs) ## {#iface-authentication-extensions-client-inputs}
```

Comment:
I think it would be good to link these to the actual definitions: `{{AuthenticationExtensionsClientInputs}}`. I tried this out myself, so I'll submit that as a PR into this one.